blog/en/introduce-milvus-2-5-full-text-search-powerful-metadata-filtering-and-more.md

```diff
@@ -0,0 +1,197 @@
+---
+id: introduce-milvus-2-5-full-text-search-powerful-metadata-filtering-and-more.md
+title: 'Introducing Milvus 2.5: Full-Text Search, More Powerful Metadata Filtering, and Usability Improvements!'
+author: Ken Zhang, Stefan Webb, Jiang Chen 
+date: 2024-12-17
+cover: assets.zilliz.com/Introducing_Milvus_2_5_e4968e1cdb.png
+tag: Engineering
+tags: Milvus, contribute to open-source projects, vector databases, Contribute to Milvus
+recommend: true
+canonicalUrl: https://milvus.io/blog/introduce-milvus-2-5-full-text-search-powerful-metadata-filtering-and-more.md
+---
+
+
+## Overview
+
+We are thrilled to present the latest version of Milvus, 2.5, which introduces a powerful new capability: _[full-text search](https://milvus.io/docs/full-text-search.md#Full-Text-Search), also known as lexical or keyword search. If you are new to search, full-text search allows you to find documents by searching for specific words or phrases within them, similar to how you search in Google. This complements our existing semantic search capabilities, which understand the meaning behind your search rather than just matching exact words.
```

Comment:
```suggestion
We are thrilled to present the latest version of Milvus, 2.5, which introduces a powerful new capability: [full-text search](https://milvus.io/docs/full-text-search.md#Full-Text-Search), also known as lexical or keyword search. If you are new to search, full-text search allows you to find documents by searching for specific words or phrases within them, similar to how you search in Google. This complements our existing semantic search capabilities, which understand the meaning behind your search rather than just matching exact words.
```